edx_exams/urls.py

```diff
@@ -30,11 +30,21 @@
 admin.autodiscover()
 
 urlpatterns = oauth2_urlpatterns + [
+    
+    # Django admin route
```

Comment:
NIT: again, not sure this comment needs to be permanent (goes for all of the comments added to this file)